packages/plugins/content/slate/src/plugins/paragraph/node.js

```diff
@@ -21,36 +21,13 @@
  */
 
 import React, { Component } from 'react'
-import { Placeholder } from 'slate'
-import { placeholder } from '../../const.js'
-import shallowEqual from 'fbjs/lib/shallowEqual'
 
 class Paragraph extends Component {
-  shouldComponentUpdate(nextProps) {
-    return !shallowEqual(this.props, nextProps)
-  }
-
-  props: {
-    children: any,
-    state: any,
-    node: any,
-    attributes: any
-  }
-
   render() {
-    const { node, state, children, attributes } = this.props
+    const { children, attributes } = this.props
     const align = this.props.node.data.get('align')
     return (
       <p {...attributes} style={{ textAlign: align }}>
-        <Placeholder
-          className="ory-plugins-content-slate-paragraph-placeholder"
```

Comment:
Particular reason for not keeping the classname?

Reply:
They removed the placeholder from slate. The placeholder is now provided as parameter to the editor.